public/tabs/chat/chat_page_content.tsx

```diff
@@ -44,12 +53,29 @@ export const ChatPageContent: React.FC<ChatPageContentProps> = React.memo((props
   const chatActions = useChatActions();
   const registry = getIncontextInsightRegistry();
   const configSchema = getConfigSchema();
+  const latestInputRef = useRef<HTMLDivElement>(null);
+  const [messageSpacerHeight, setMessageSpacerHeight] = useState(0);
 
   useLayoutEffect(() => {
-    if (!props.chatScrollTopRef.current) {
+    if (latestInputRef.current) {
+      setMessageSpacerHeight(
```

Comment:
Shall we setMessageSpacerHeight when loading is true?

Reply:
`latestInputRef` only attach to a element when loading is true, but good to also add a double check here for clarification anyway.